pom.xml

```diff
@@ -89,10 +89,10 @@
             <artifactId>xembly</artifactId>
             <version>0.21</version>
         </dependency>
-        <dependency>
-            <groupId>com.jcabi</groupId>
-            <artifactId>jcabi-manifests</artifactId>
-        </dependency>
+<!--         <dependency> -->
```

Comment:
I fixed it locally.
I will commit when solving all issues.
Fighting with RtDatatypeConverter checkstyle :(